src/main/java/com/develhope/spring/vehicles/entity/Scooter.java

```diff
@@ -0,0 +1,24 @@
+package com.develhope.spring.vehicles.entity;
+
+public class Scooter implements Vehicles {
+    String brand;
+    String model;
+    int engineSize;
+    String color;
+    int power;
+    String gearbox;
+    int registrationYear;
+    //Fueltype fuelType; bisogna creare classe Fueltype
+    double price;
+    double discount;
+    String accesories;
+    boolean isNew;
+    //VehicleState vehicleState; bisogna creare classe VehicleState
+    boolean purchasable;
```

Comment:
Fabri, questi due ultimi, per adesso non li stiamo aggiungendo, forse li implementiamo in un altro modo

Reply:
quindi vanno eliminati:
- VehicleState vehicleState;
-     boolean purchasable;
giusto?

Reply:
i due booleani finali
quindi purchasable e rentable

Reply:
fai un pull prima di pushare

Reply:
okok mi ero semplicemente rifatto allo schema su lucid.app, fatto